project-document/docs/domain-model/group-context/food.md

```diff
@@ -2,6 +2,11 @@
 
 ## Food (Entity)
 
+
+### Unit (Value Object)
+
+The lenghth of unit must be less than equal to 10 characters.
```

Comment:
Is the length less than equal to 10 instead 20?

Reply:
Thank you so much for mentioning it. I totally overlooked it.